systemvm/patches/debian/config/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -96,7 +96,7 @@ def _redundant_on(self):
                 d = s.replace(".templ", "")
             CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, s), "%s/%s" % (self.CS_ROUTER_DIR, d))
         CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, "keepalived.conf.templ"), self.KEEPALIVED_CONF)
-        CsHelper.copy_if_needed("%s/%s" % (self.CS_TEMPLATES_DIR, "conntrackd.conf.templ"), self.CONNTRACKD_CONF)
+        CsHelper.copy("%s/%s" % (self.CS_TEMPLATES_DIR, "conntrackd.conf.templ"), self.CONNTRACKD_CONF)
```

Comment:
Hi Daan,

This will be needed only in case of redundant router. The current code tries to copy this file only if it is a rvr. It will not be copied for normal routers.